File: 김정현/Prac_4/BOJ_1546.java
package 김정현.Prac_4;

import java.util.Scanner;

public class BOJ_1546 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int N = scanner.nextInt();

        int score[] = new int[N];
    
        double result = 0.0, sum = 0, avg = 0;

        for(int i = 0; i < N; i++){
            score[i] = scanner.nextInt();
        }

        int max = score[0];

        for(int k = 0; k < N; k++){
            sum += score[k];
            
            if(max < score[k]){
                max = score[k];
            }
        }
        avg = sum / N;
        result = avg/max*100;
       
        System.out.println(result);
       
	}  
}

File: 지석민/BOJ_1546.c
#define _CRT_SECURE_NO_WARNINGS
#include <stdio.h>
int main(void) {
   
    int a = 0;
    int score[1001];
    int max = 0;
    double avg = 0;

    scanf("%d", &a);

    for (int i = 0; i < a; i++) {
        scanf("%d", &score[i]);
        if (max < score[i]) {
            max = score[i];
        }
    }

    for (int j = 0; j < a; j++) {
        
        avg = avg + (((double)score[j] / max) * 100);

    }

    printf("%f", avg / a);
    

    return 0;
}